phalcon/mvc/model/manager.zep

```diff
@@ -2249,4 +2235,157 @@ class Manager implements ManagerInterface, InjectionAwareInterface, EventsAwareI
 
 		return success;
 	}
+
+	/**
+	 * Checks if a record already exists or not
+	 */
+	public function exists(<ModelInterface> model) -> boolean
```

Comment:
Well this can be in manager maybe, but i think the rest of `_anyMethod` from model should go to new class.